package.json

```diff
@@ -26,7 +26,7 @@
     "eth-block-tracker": "^4.4.2",
     "eth-json-rpc-errors": "^2.0.2",
     "eth-json-rpc-filters": "^4.2.1",
-    "eth-json-rpc-infura": "^4.0.1",
+    "eth-json-rpc-infura": "5.1.0",
```

Comment:
Was this meant to be pinned on `5.1.0` specifically? :thinking:

Reply:
Boooo. No it was not. I'm trying to sneak in an unpinning here: https://github.com/MetaMask/web3-provider-engine/pull/357